### PR TITLE
Expanded candidate image fix, Todd Gloria centered, and raised amount formatting

### LINKED ISSUE #109: Enhancement Req: Add candidate image to detail section
There's a fair amount of white space above the candidate's name in the detail section. It would be helpful to make it more clear which candidate the details belong to by showing their image above their name. 

Screenshot:
Screen Shot 2020-09-08 at 9 41 07 PM

Testing Env: Chrome Version 84.0.4147.89 (Official Build) (64-bit) on Mac OS 10.15.6 (19G73)

### PR DESCRIPTION
**Pull request by domdreams**
Closes #109 , #94 and #92

### CODE REVIEW

**Comment by domdreams** [comment]
Fixed the candidate card images, formatting the raised amount, and fixed the percent value on the expanded card.

**Comment by gSoD17** [comment]
Hi Dom, looking through your changes and the other pull requests and everything is looking good so far, I believe that Robert's pr #112 has also resolved the percentage fixes as well.  And did you pull Aidan's fixes with the charts updating?

**Comment by domdreams** [comment]
Hey @gSoD17 I saw robert's pr after I did mine. Up to you on what you want me to do. I did not pull Aidans changes. What should I do about that?

**Comment by gSoD17** [comment]
Hi @domdreams, I believe that we should go with Robert's changes and that for Aidan's changes, I just wanted to know if you have pulled it yet from master since he solved the charts issue.

**Comment by domdreams** [comment]
@gSoD17 Ok sorry to do duplicate work. Do I need to change my code on this pull request? I am not sure how to proceed with the two conflicts.

**Comment by robertgz** [comment]
@domdreams @gSoD17  I'll go ahead and merge my changes for #111 and #112 into the dev branch.

**Comment by robertgz** [comment]
I will try to resolve the conflict.

File: src/app/components/candidate-card-expanded/candidate-card-expanded.component.html [hunk 1]
```diff
@@ -103,7 +103,7 @@ <h2>{{ candidate['description'] }}</h2>
   
         <ng-container matColumnDef="percentage">
           <th mat-header-cell *matHeaderCellDef>Percentage</th>
-          <td [ngClass]="{'remove-border': i == dataSource.data.length - 1}" mat-cell *matCellDef="let element; let i = index">{{ element.percentage | percent }}</td>
+          <td [ngClass]="{'remove-border': i == dataSource.data.length - 1}" mat-cell *matCellDef="let element; let i = index">{{ element.percentage/100 | percent }}</td>
```
Comment (robertgz) [inline src/app/components/candidate-card-expanded/candidate-card-expanded.component.html:106]:
```suggestion
          <td [ngClass]="{'remove-border': i == dataSource.data.length - 1}" mat-cell *matCellDef="let element; let i = index">{{ element.percentage | number:'1.0-0' }}%</td>
```

This change should resolve the merge conflict.

**Pull request merged**